### ISSUE TITLE
Design a Mockup for FlashFile UI

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#29]
## User story
1. As a user 
2. I want to see a visual representation of the FlashFile UI
3. So that I understand how to select, validate, and view the information about flash files

## Acceptance criteria
* Visual representations of file selection, validation feedback, and file information display is included
* User feedback is also reflected in the mockup
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by AndreasFreund** [comment]
Maybe add a progress bar for the flashing progess
